signbank/dictionary/templates/dictionary/gloss_detail.html

```diff
@@ -862,7 +862,8 @@ <h2 id='modalTitleDelete'>{% trans "Delete This Sign" %}</h2>
             {% for lang, annotation_idgloss_translation in annotation_idgloss.items %}
             <tr>
                 <th>
-                    {% trans "Annotation ID Gloss" %} ({{lang}})
+                    {% trans "Annotation ID Gloss" %}
+                    {% if SHOW_DATASET_INTERFACE_OPTIONS %} ({{ lang }}) {% endif %}
```

Comment:
I am wondering whether `{% if lang %}` would also work and whether that would be better? Of perhaps `{% if SHOW_DATASET_INTERFACE_OPTIONS and lang %}`?